pybytes/pytrack/boot.py

```diff
@@ -0,0 +1 @@
+# boot.py -- run on boot-up
```

Comment:
Could you remove the `boot.py` since it's empty?

Reply:
Removed empty boot.py file.

Reply:
Added information in readme.md file.